Microsoft.Toolkit.Uwp.UI.Controls/TabView/TabViewItem.cs

```diff
@@ -72,7 +72,7 @@ protected override void OnPointerPressed(PointerRoutedEventArgs e)
                     _isMiddleClick = true;
                 }
 
-                // Disable unwanted behaviour inherited by ListViewItem:
+                // Disable unwanted behavior inherited by ListViewItem:
```

Comment:
inb4 English devs protesting 🤣

Reply:
Yeah, this is probably a nit-pick, but we do use 'Behaviors' a lot in XAML, so we should be consistent with that at least! 😋